src/Editor.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+
+const replaceHTMLMap = {
+    "&": "&amp;",
+    "<": "&lt;",
+    ">": "&gt;",
+};
+
+function escape(given: string): string {
+    return given.replace(/[<>&]/g, c => replaceHTMLMap[c] || "");
+}
+
```

Comment:
Maybe we should move this function to a separate module. This can be a separate pull request, though. In that case we should also escape quotes (' and ").